apps/somnia-bot/project.json

```
@@ -0,0 +1,77 @@
+{
+  "name": "somnia-bot",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "apps/somnia-bot/src",
+  "projectType": "application",
+  "targets": {
+    "build": {
+      "executor": "@nx/esbuild:esbuild",
+      "outputs": ["{options.outputPath}"],
+      "defaultConfiguration": "production",
+      "options": {
+        "platform": "node",
+        "outputPath": "dist/apps/somnia-bot",
+        "format": ["cjs"],
+        "bundle": false,
+        "main": "apps/somnia-bot/src/main.ts",
+        "tsConfig": "apps/somnia-bot/tsconfig.app.json",
+        "assets": ["apps/somnia-bot/src/assets"],
```

```suggestion
        "assets": [
          "apps/somnia-bot/src/assets",
          {
            "input": "./apps/somnia-bot/src",
            "glob": "config.json",
            "output": "apps/somnia-bot/src/"
          }
        ],
```
Looking at `dist/apps/somnia-bot`, I noticed `config.json` wasn't being copied there. Need to add non-code items as assets here
https://stackoverflow.com/questions/71015454/nx-copy-one-extra-file-to-build-directory
https://nx.dev/recipes/tips-n-tricks/include-assets-in-build